content/handbook/support/workflows/confirmation_emails.md

```
@@ -0,0 +1,192 @@
+このワークフローは、ユーザーが確認メールを受信していないと述べているケースを扱います。確認メール内のアクティベーショントークンは24時間のみ有効です。それを過ぎると、ユーザーは新しい確認メールが必要となります。
+
+## **Stage 0:** チケットのトリアージ
+
+チケットに対応する前に、`SaaS Account` フォームと `Did not receive confirmation email` 問題タイプで正しくトリアージされていることを確認し、
+[SaaS Account Ticket Helper](/handbook/security/customer-support-operations/zendesk/apps/global#gitlab-super-app) アプリケーションがアクティブになるようにします。
+
+ユーザーが正しい問題タイプをすでに選択している場合、エージェントが初めてチケットを開いたときに自動化がアクティブになります。何らかの理由で SaaS Account Ticket Helper アプリケーションが問題を解決できなかった場合は、以下のセクションの手順に従って手動で解決します。
+
+## **Stage 1:** アカウントの特定
+
+問題を解決する前に、まず該当するアカウントを特定する必要があります。これは、[GitLab User Lookup App](#method-1-check-gitlab-user-lookup-app) または [GitLab Admin](#method-2-check-gitlab-admin) のいずれかを確認することで実行できます。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)